javascript/react/src/components/StepForm/selectStepForm.ts

```diff
@@ -94,7 +94,7 @@ const forms = {
 export function selectStepForm(
 	step: Step<unknown>,
 ): React.FC<StepFormProps<unknown>> {
-	const result: React.FC<StepFormProps<any>> = get(forms, step.verb)
+	const result: React.FC<StepFormProps<any>> | undefined = get(forms, step.verb)
```

Comment:
Is the expectation here that finding a workflow verb instance will throw on the client?

Reply:
The `get` interface can possibly return undefined, and the clause directly below this checks for null/undefined. 
This looks like a case of "I'm not sure how this passed CI to begin with"

Reply:
Oh interesting, I thought you were explicitly supporting undefined as a return because there is no verb form for workflow. In the tagged verbs util we have a little test to ensure each verb has an entry; maybe we need something like this for the forms - theoretically `undefined` shouldn't be possible because that will invariably break the UI somewhere.